src/types/platforms-types/mintsongs-v2.ts

```diff
@@ -70,7 +68,7 @@ const mapArtistProfile = ({ apiTrack, nft, contract }: { apiTrack: any, nft?: NF
 
 const mapNFTtoLatestTrackID = (nft: NFT, dupNFTs: NFT[]): string => {
   const primaryNFT = selectPrimaryNFTForTrackMapper(dupNFTs);
-  return `ethereum/${formatAddress(primaryNFT.contractAddress)}/${primaryNFT.tokenId}`;
+  return ethereumTrackId(primaryNFT.contractAddress, String(primaryNFT.tokenId));
```

Comment:
minor comment/point to be aware of:

Since tokenId is a bigint, the more canonical way to convert it would be using toString()

It probably won't make a difference in our case, but for reference: https://stackoverflow.com/questions/3945202/whats-the-difference-between-stringvalue-vs-value-tostring

```suggestion
  return ethereumTrackId(primaryNFT.contractAddress, primaryNFT.tokenId.toString());
```